atest/robot/cli/dryrun/executed_builtin_keywords.robot

```diff
@@ -13,3 +13,11 @@ Set Library Search Order
     Should Be Equal    ${tc.kws[1].name}    Second.Parameters
     Should Be Equal    ${tc.kws[2].name}    First.Parameters
     Should Be Equal    ${tc.kws[4].name}    Dynamic.Parameters
+
+Set Tags
+    ${tc} =    Check Test Case    ${TESTNAME}
```

Comment:
This is actually unnecessary as `Check Test Tags` uses `Check Test Case` internally. I can remove these myself.

Reply:
I was referring to the existing tests and thought it was the convention. Thanks for the information 😃